common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -41,35 +62,39 @@ All of the above are repeated, independently, for the second notch and are prefa
 
 .. note:: only one filter can be mode 4(FFT).
 
+.. _notch_center_freq:
+
 
 Determining Notch Filter Center Frequency
 =========================================
 
-Before actually setting up a dynamic notch filter, the frequencies that are desired to be rejected must first be determined. This is crucial if :ref:`common-throttle-based-notch` is used. While the other methods do not require this knowledge apriori, its still worthwhile as a comparison point for the post filter activation analysis of the filter's effectiveness
+Before actually setting up a dynamic notch filter, the frequencies that are desired to be rejected must first be determined. This is crucial if a static notch or :ref:`common-throttle-based-notch` is used. While the other methods do not require this knowledge apriori, its still can worthwhile as a comparison point for the post filter activation analysis of the filter's effectiveness.
```

Comment:
We should probably replace "apriori" with "beforehand".  Lots of non-native-english speakers in the world.